diff --git a/Dockerfile b/Dockerfile
index acddb67..9f345e7 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -9,10 +9,10 @@ COPY docker/startup.sh /bin/
 COPY docker/test.sh /bin/

 # Set execute permissions
-RUN chmod +x /bin/startup.sh
-RUN chmod +x /bin/test.sh
+RUN chmod +x /bin/startup.sh /bin/test.sh

 WORKDIR /data
 VOLUME /data
 EXPOSE 587
-ENTRYPOINT startup.sh
+USER 65532:65532
+ENTRYPOINT ["/bin/startup.sh", "node", "/bin/smtp2graph.js"]
